src/GLdispatch/vnd-glapi/entry_files.mk

```diff
@@ -37,6 +37,18 @@ MAPI_GLDISPATCH_ENTRY_FILES = entry_aarch64_tsd.c
 MAPI_GLDISPATCH_ENTRY_FILES += entry_common.c
 endif
 
+if GLDISPATCH_TYPE_PPC64LE_TSD
+MAPI_GLDISPATCH_ENTRY_FILES = entry_ppc64le_tsd.c
+MAPI_GLDISPATCH_ENTRY_FILES += entry_x86_64_common.c
+MAPI_GLDISPATCH_ENTRY_FILES += entry_common.c
+endif
+
+if GLDISPATCH_TYPE_PPC64LE_TLS
+MAPI_GLDISPATCH_ENTRY_FILES = entry_ppc64le_tls.c
+MAPI_GLDISPATCH_ENTRY_FILES += entry_x86_64_common.c
```

Comment:
It might be nice to rename this file at some point and make the ARM version the special case.